src/OnlineStats.jl

```diff
@@ -47,7 +47,7 @@ export
     OrderStats,
     Part, Partition, PlotNN, ProbMap, P2Quantile,
     Quantile,
-    ReservoirSample,
+    ReservoirSample, LogSumExp,
```

Comment:
Exports are alphabetical.  That's my only (very minor) comment for this PR

Reply:
Ah, ok I didn't notice that. Should be a quick fix :)